chapters/os_tooling.md

```diff
@@ -0,0 +1,85 @@
+# Offline Compilation of OpenCL Kernel Sources
```

Comment:
Would it be simpler to call this page `Offline Compilation of OpenCL Kernel Sources to SPIR-V`.

Compiling to binaries only gets 2-3 sentences, the rest is about SPIR-V.   I'm not sure how to write more about compiling to binaries as it's all vendor specific tools.

This is just an option to consider.

Reply:
I think I would prefer to leave it more general. For example, clang can compile to some binaries i.e. AMDGPU and I am linking in a page where it is described although I appreciate it is pretty trivial.

Reply:
Okay.